News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderListActivity.java

```diff
@@ -391,7 +391,7 @@ private void initAccountManager() {
 			Account account = new Account(getString(R.string.app_name), accountType);
 			mAccountManager.addAccountExplicitly(account, "", new Bundle());
 
-			SyncIntervalSelectorActivity.setAccountSyncInterval(this, mPrefs);
+			SettingsFragment.setAccountSyncInterval(this, R.integer.default_sync_minutes);
```

Comment:
Are you sure that this shouldn't be `getResources().getInteger(R.integer.default_sync_minutes)` or something like that?